mathesar/api/serializers/columns.py

```diff
@@ -0,0 +1,101 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import empty
+from rest_framework.settings import api_settings
+
+
+class InputValueField(serializers.CharField):
+    """
+    Takes in an arbitrary value. Use to emulate our column and record creation and
+    update endpoints, which handle arbitrary data pulled from request.data
```

Comment:
Is this comment accurate?  I note that this is only used in this column serializer file.  If it's intended to be used for records as well, consider moving it to a `util` file.

Reply:
It is not used elsewhere. I copied the comment as-is, I'll fix the comment in a future PR.